evm/src/main/java/org/hyperledger/besu/evm/operation/SelfDestructOperation.java

```diff
@@ -60,19 +75,22 @@ public OperationResult execute(final MessageFrame frame, final EVM evm) {
     final Address address = frame.getRecipientAddress();
```

Comment:
Perhaps a little verbose, but while reviewing, I renamed this to `addressOfAccountToMaybeDestruct`

Reply:
Spec has `originator`, so I went with that.